test/account-creation.js

```diff
@@ -59,45 +57,36 @@ describe('Identity Provider', function () {
     })
   })
 
-  describe('generating a certificate', function () {
+  describe('generating a certificate', () => {
     beforeEach(function () {
       rm('accounts/nicola.localhost')
     })
     after(function () {
       rm('accounts/nicola.localhost')
     })
 
-    it('should generate a certificate if spkac is valid', function (done) {
+    it('should generate a certificate if spkac is valid', (done) => {
       var spkac = read('example_spkac.cnf')
       var subdomain = supertest.agent('https://nicola.' + host)
       subdomain.post('/api/accounts/new')
-        .send('username=nicola')
-        .expect(200)
-        .end(function (err, req) {
-          if (err) return done(err)
-
-          subdomain.post('/api/accounts/cert')
-            .send('spkac=' + spkac + '&webid=https%3A%2F%2Fnicola.localhost%3A3457%2Fprofile%2Fcard%23me')
-            .expect('Content-Type', /application\/x-x509-user-cert/)
-            .expect(200)
-            .end(done)
-        })
-    })
+        .send('username=nicola&spkac=' + spkac)
+        .expect('Content-Type', /application\/x-x509-user-cert/)
+        .expect(200, done)
+    }).timeout(20000)
```

Comment:
> I mean, the answer could be as simple as 'I have too many apps running on my laptop'. :)

I'm not so sure that a teeny tiny single-threaded web server concerned with teeny tiny file IO would be slowed down too much by other apps on the system... :wink:

> But I suspect it's the cert generation...

I didn't realize the server did any cert generation. I thought the client was responsible for that.

> ...and the individual creation of each account resource via rdflib. (Which, if that's true, this should be faster now, as of this PR)

Right, because this server now does that work when it's first booted up.

How about removing the timeouts to verify your claim?